### PR TITLE
SAK-46178 Rubrics: Added a couple of index names

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-46178

### CODE REVIEW

File: rubrics/api/src/main/java/org/sakaiproject/rubrics/api/model/Evaluation.java [hunk 1]
```diff
@@ -62,10 +61,10 @@
 @NoArgsConstructor
 @Table(name = "rbc_evaluation",
     indexes = { @Index(name = "rbc_eval_owner",  columnList="ownerId"),
```
Comment (bjones86) [inline rubrics/api/src/main/java/org/sakaiproject/rubrics/api/model/Evaluation.java:63]: So this columnList still references `ownerId`. Could that be why auto.ddl is bringing it back?

Comment (adrianfish) [reply rubrics/api/src/main/java/org/sakaiproject/rubrics/api/model/Evaluation.java:63]: I'm stumped

Comment (bjones86) [reply rubrics/api/src/main/java/org/sakaiproject/rubrics/api/model/Evaluation.java:63]: Interesting. I'm not sure how to explain this either, and I don't see how naming some indexes would make a difference.

I can try applying this PR to my local instance to see if it fixes it.

Comment (adrianfish) [reply rubrics/api/src/main/java/org/sakaiproject/rubrics/api/model/Evaluation.java:63]: Works fine on my local mysql, no ownerId column on rbc_tool_item_rbc_assoc with auto ddl on.

Comment (bjones86) [reply rubrics/api/src/main/java/org/sakaiproject/rubrics/api/model/Evaluation.java:63]: I deployed this code and `ownerId` remained. I then manually removed the column and restarted my server with this code deployed and `ownerId did not come back. I then cleaned my Tomcat, redeployed master (without these changes) to check if `ownerId` came back again: it did not.

I'm not sure what the problem was. Maybe old artifacts in Tomcat? Anyway, it doesn't seem like the column is coming back anymore.

**Review by bjones86 — APPROVED** [review]

**Pull request merged**